javascript/javascript2/week3/js2-week3-Homework/fastestPresser/fastestPresser.js

```diff
@@ -0,0 +1,71 @@
+"use strict"
+
+const timeInput = document.querySelector('#numberInput');
+const gameStart = document.querySelector('#btn1')
+const playerS = document.querySelector('.player-S')
+const playerL = document.querySelector('.player-L')
+const result = document.querySelector('.result')
+const restart = document.querySelector('.restart')
+const newUlTag = document.createElement('ul')
+const ulTag = document.createElement('ul')
+playerS.appendChild(newUlTag)
+playerL.appendChild(ulTag)
+
+let counterL = 0;
+let counterS = 0;
+
+//Start game getting timeInput from players.. rendering result after time up
+function beginGame () {
+    setTimeout(()=>{
+if(!timeInput.value){
+    alert('please enter time to play game')
+    !gameResult
+}else alert('Game over')
```

Comment:
this looks weird, I would suggest to always use `{` and `}`, especially when the `if` uses curly brackets then the `else` should to :)

`alter` is very old fashioned javascript. Dont use it. use the DOM to show the user that the game is over.